packages/block-editor/src/components/inserter/search-results.native.js

```diff
@@ -104,8 +103,7 @@ function InserterSearchResults( {
 					...listProps.contentContainerStyle,
 					{
 						paddingBottom:
-							( safeAreaBottomInset ||
-								styles.list.paddingBottom ) + searchFormHeight,
+							safeAreaBottomInset || styles.list.paddingBottom,
```

Comment:
We no longer need to add the search form height here because this value is now included in the calculations of the `maxHeight`, which is part of the style prop `contentContainerStyle`.